src/main.js

```diff
@@ -35,112 +35,118 @@ startButton.addEventListener('click', startCountDown);
 logButton.addEventListener('click', logActivity);
 createNewActivityBtn.addEventListener('click', createNewActivity);
 
-function changeColor(button1, category1, button2, category2, button3, category3) {
-    if (button2 == undefined) {
-        button1.classList.add(category1);
-    } else if (button3 == undefined) {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-    } else {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-        button3.classList.remove(category3);
-    }
+function addClassProperty(button1, category1, button2, category2) {
+  if (!button2) {
+    button1.classList.add(category1);
+  } else {
+    button1.classList.add(category1);
+    button2.classList.add(category2);
+  }
+}
+
+function removeClassProperty(button1, category1, button2, category2) {
```

Comment:
Same comment as `addClassProperty` - this function signature could be simplified and called repeatedly if needed for multiple elements.